smoke-tests/smoke-tests.gradle

```
@@ -1,3 +1,8 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_11
+  maxJavaVersionForTests = JavaVersion.VERSION_11
```

👍
just want to add a comment here to avoid confusion
```suggestion
  // we only need to run the Spock test itself under a single Java version, and the Spock test in
  // turn is parameterized and runs the test using different docker containers that run different
  // Java versions
  minJavaVersionForTests = JavaVersion.VERSION_11
  maxJavaVersionForTests = JavaVersion.VERSION_11
```